packages/lit-dev-content/site/docs/v3/releases/upgrade.md

```diff
@@ -1,40 +1,63 @@
 ---
-title: Upgrade guide
+title: Upgrade to Lit 3.x guide
```

Comment:
nit: wdyt about titling this "Lit 3 upgrade guide" ?
I think just listing the major version looks better than adding ".x"

Reply:
Love it!